components/Events/Filters.js

```diff
@@ -0,0 +1,142 @@
+import {faChevronUp, faChevronDown} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useState, React } from 'react';
+import events from '../../data/event';
+import styles from '../../styles/components/Events/Filters.module.scss';
+
+
+export default function filters(props) {
+	const days = ['Sunday','Monday','Tuesday','Wednesday','Thursday','Friday','Saturday'];
```

Comment:
Can we do the same as L13? Basically same idea + code consistency + easier to read through code / debug if we can print out `chosenDays`